lib/kubeclient/common.rb

```diff
@@ -24,6 +24,8 @@ module ClientMixin
       ssl_socket_class: nil
     }.freeze
 
+    DEFAULT_HTTP_PROXY_URI = nil
+
     attr_reader :api_endpoint
     attr_reader :ssl_options
     attr_reader :auth_options
```

Comment:
please add the http proxy to the exposed attributes. I'ts quite handy for different use cases like debugging or when users of the client want to know of these properties. (I think we better add socket_options in a different patch)